### ISSUE TITLE
unknown input format error

### ISSUE THREAD

**Issue by cdkodira** [gambalab/DREEP#2]
I downloaded your sample data set. When I ran this code , I get error saying the format is unknown.
Any help?
Dilip
dreep.data <- DREEP::runDREEP(M = data$gficf,
+                               n.markers = 250,
+                               gsea = "multilevel",
+                               gpds.signatures = c("CTRP2","GDSC"), th.pval = 0.05, verbose=T)
14:55:20 Loading GPDS signatures..
Error in readRDS(paste0(find.package("DREEP"), "/data/", i, ".gpds.rds")) : 
  unknown input format

**Comment by gambalab** [comment]
unfortunately is a windows related problem, that for now I did not have time to investigate yet

**Comment by hmumme** [comment]
@cdkodira,
I got this error as well, when trying the example on a Linux system. 

> Error in readRDS(paste0(find.package("DREEP"), "/data/", i, ".gpds.rds")) :
unknown input format

Upon inspection, I noticed the files in the `DREEP/data` directory may not have been downloaded correctly when installing. The files installed were around 100 B while those housed in the github are over 60 MiB. 

If you download the databases of interest directly from here - https://github.com/gambalab/DREEP/tree/main/data - and replace them in your library folder - something like `/usr/lib/__/DREEP/data` - then it should run.

> System Details:
> R version 4.1.2 (2021-11-01)
> Platform: x86_64-redhat-linux-gnu (64-bit)
> Running under: Rocky Linux 8.7 (Green Obsidian)_
